yt/frontends/gadget/fields.py

```diff
@@ -75,7 +75,8 @@ def _temperature(field, data):
                 x_H = 0.76
                 gamma = 5.0/3.0
                 a_e = data[ptype, 'ElectronAbundance']
-                mu = 4.0 / (3.0 * x_H + 1.0 + 4.0 * x_H * a_e)
+                # mu = 4.0 / (3.0 * x_H + 1.0 + 4.0 * x_H * a_e)
+                mu = 4.0 / (3.0 + x_H + 4.0 * x_H * a_e)  # this should be correct
```

Comment:
Here and below please delete the commented old code. In general we don't like leaving commented out code behind because it makes the code harder to understand. If we ever need to go back to understand how the code evolved we have the repository. You can also delete the "this should be correct" comments.